packages/components/date-picker/src/date-picker-com/panel-date-pick.vue

```diff
@@ -512,9 +512,10 @@ export default defineComponent({
       }
     }
 
-    const isValidValue = (date_) => {
+    const isValidValue = (date: Dayjs) => {
+      if (!(date instanceof dayjs)) return false
       return (
-        date_.isValid() && (disabledDate ? !disabledDate(date_.toDate()) : true)
+        date.isValid() && (disabledDate ? !disabledDate(date.toDate()) : true)
       )
```

Comment:
```ts
import { Dayjs } from 'dayjs'
const isValidValue = (date: unknown) => {
  return (
    date instanceof Dayjs &&
    date.isValid() &&
    (disabledDate ? !disabledDate(date.toDate()) : true)
  )
}
```

Reply:
If so, the `Dayjs` type need to remove
https://github.com/element-plus/element-plus/blob/e5025ce50deb167017b6c3b7c11eb8d3bfbb0d5a/packages/components/date-picker/src/date-picker-com/panel-date-pick.vue#L191

Reply:
Yes.